src/SOS/SOS.Package/SOS.Package.csproj

```diff
@@ -0,0 +1,120 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Microsoft.NETCore.SOS</AssemblyName>
```

Comment:
One simple question here. The package will be called `Microsoft.NETCore.SOS.3.0.0-preview7.PATCH.REV` while the manifest will have `3.0.PATCH_HI.PATCH_LO`. Does this matter for the store? As in, does the version in the manifest need to match the package version?

Reply:
This shouldn't matter.  The version in the manifest is used by the extension gallery.  Their build system will use the package version to "install" or whatever.  Basically the debuggers will download/load the latest version according to the manifest.